CMakeLists.txt

```diff
@@ -9,6 +9,19 @@ project(
     VERSION 0.0.1
 )
 
+# gersemi: off
```

Comment:
If more than 60% of the CMake file is ignored, we might as well remove the cmake formatter.

Reply:
There are other cmake files.

Reply:
I'm pro removing the formatter, it's too wrong too often to really be useful